server/server.go

```diff
@@ -82,6 +82,10 @@ func New(url string) *RPCServer {
 	return &server
 }
 
+func Close() {
```

Comment:
What's the plan here?

Reply:
Close should close stop consuming new messages.

Reply:
Hmm, I might be missing something but when we starting the listener in `listenAndServer()` we already makes sure that we cleanly calls close on the connection and on the channel (which is where we consume messages):

```
// Get connection
defer conn.Close()

// Get input channel
defer inputCh.Close()

// Get output channel
defer outputCh.Close()
```

The only thing we needs to make sure is that the responder is finished (which is reading from an "internal" channel). So, `close(s.responses)`? Or should we store all closable references and handle them here?